build_forklift_bazel.sh

```diff
@@ -17,11 +17,34 @@ cd ${FORKLIFT_DIR:-forklift}
 
 export REGISTRY=localhost:5001
 export REGISTRY_TAG=latest
-export REGISTRY_ACCOUNT=""
+# REGISTRY_ACCOUNT cannot be empty with docker
+export REGISTRY_ACCOUNT="ci"
+
+cat << __EOF__ > /etc/containers/registries.conf.d/localhost.conf
+[[registry]]
+location = "${REGISTRY}"
+insecure = true
+__EOF__
+
+
+#export CONTAINER_CMD="/usr/bin/docker"
+
+make push-ovirt-populator-image
+make push-populator-controller-image
```

Comment:
bazel run push-populator-controller

Reply:
i'd like to try to make it work with `make` to avoid bazel duplication.